aidial_adapter_vertexai/chat/errors.py

```diff
@@ -63,13 +60,7 @@ def __init__(self, message: str):
         super().__init__(self.message)
 
     def to_dial_exception(self) -> DialException:
-        return DialException(
-            status_code=422,
-            type="invalid_request_error",
-            message=self.message,
-            code="invalid_argument",
```

Comment:
`code` parameter will change from "invalid_argument" to "400" after this change

Reply:
good catch, fixed